java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/models/VariableMirrorTranslator.java

```diff
@@ -156,17 +156,7 @@ private static Object createPristineInstanceOf(Class clazz) {
             Object newInstance = constructor.newInstance();
             return newInstance;
             
-        } catch (InstantiationException ex) {
-            logger.log(Level.INFO, "Problem while creating a pristine instance of "+clazz, ex);
-        } catch (IllegalAccessException ex) {
-            logger.log(Level.INFO, "Problem while creating a pristine instance of "+clazz, ex);
-        } catch (IllegalArgumentException ex) {
-            logger.log(Level.INFO, "Problem while creating a pristine instance of "+clazz, ex);
-        } catch (InvocationTargetException ex) {
-            logger.log(Level.INFO, "Problem while creating a pristine instance of "+clazz, ex);
-        } catch (NoSuchMethodException ex) {
-            logger.log(Level.INFO, "Problem while creating a pristine instance of "+clazz, ex);
-        } catch (SecurityException ex) {
+        } catch (InstantiationException | IllegalAccessException | IllegalArgumentException | InvocationTargetException | NoSuchMethodException | SecurityException ex) {
```

Comment:
could be further collapsed to `(ReflectiveOperationException | rest...)` since ReflectiveOperationException is the supertype of most reflection related exceptions.

Reply:
Hey @mbien , if you like this change, I'll do it..
```
        } catch (ReflectiveOperationException | IllegalArgumentException | SecurityException ex) {
            logger.log(Level.INFO, "Problem while creating a pristine instance of "+clazz, ex);
        }
```